### ISSUE TITLE
Multi cursor click has broken again.

### ISSUE THREAD

**Issue by brentonstrine** [atom/atom#20987]
This is a regression of https://github.com/atom/atom/issues/20803. 

### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: <https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode>
    * Followed all applicable steps in the debugging guide: <https://flight-manual.atom.io/hacking-atom/sections/debugging/>
    * Checked the FAQs on the message board for common solutions: <https://discuss.atom.io/c/faq>
    * Checked that your issue isn't already filed: <https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom>
    * Checked that there is not already an Atom package that provides the described functionality: <https://atom.io/packages>

### Description
Multi cursor clicking has broken. 

### Steps to Reproduce
1. Hold command (or control) and click somewhere. 

**Expected behavior:**

1. The new clicked location spawns another cursor. 

**Actual behavior:**
1. The cursor moves from the old location to the new location.

**Reproduces how often:**

This always happens. 

### Versions

Atom 1.48.0
MacOS 10.15.5

### Additional Information

This can be temporarily solved by editing `config.cson` to the following:

```
"*":
  core:
    editor:
      multiCursorOnClick: true
  editor:
    multiCursorOnClick: true
```

**Comment by UziTech** [comment]
fixed by #20804

**Comment by UziTech** [comment]
see https://github.com/atom/atom/pull/20804#issuecomment-643279418 for a temporary fix
